src/inverted-index.js

```diff
@@ -57,8 +55,7 @@ class invertedIndex {
   isValidFileName(fileName) {
     if (fileName.match('.json$') === null) {
```

Comment:
The $ helps check for a value at the end of a certain string, so it enables me to check for '.json' at the  end of a file name